cli/jobs/automl-standalone-jobs/cli-automl-distributed-regression-task-hardware-performance/cli-distributed-regression.yaml

```diff
@@ -0,0 +1,48 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/preview/0.0.1/autoMLJob.schema.json
+type: automl
+
+experiment_name: cli_regression_task_distributed_lightgbm
+# name: dpv2-cli-regression-train-job-01
+description: A regression job using hardware performance dataset
+
+compute: azureml:automl-distributed-cluster
+
+task: regression
+log_verbosity: debug
+primary_metric: r2_score
+
+target_column_name: "ERP"
+
+training_data:
+  path: "./data/training-mltable-folder"
+  type: mltable
+
+limits:
+  timeout_minutes: 600
+  max_trials: 5
+  max_concurrent_trials: 4
+  trial_timeout_minutes: 120
+  enable_early_termination: true
+  max_nodes: 6
+
+# This is example of custom featurization
+# If you don't have this section, AutoML will default to automatic featurization
+featurization:
+  mode: custom
+  blocked_transformers: [LabelEncoder]
```

Comment:
Blocked transformers are ignored for distributed training since we only support one transformer per type (numerical, categorical, text). Hence let us remove blocked_transformers here

https://msdata.visualstudio.com/Vienna/_git/AzureMlCli?path=/src/azureml-automl-runtime/azureml/automl/runtime/_ml_engine/featurizer_suggestion/static_suggestions.py&version=GBmaster&line=137&lineEnd=139&lineStartColumn=8&lineEndColumn=85&lineStyle=plain&_a=contents

Reply:
On a second thought we should we may want to remove custom featurization part from this notebook to make sure that notebook is laser focused on distributed featurization and training.

Reply:
do this for both classifiacation/regression

Reply:
Thank you. Removed custom featurization from regression notebook. Classification notebook does not have no custom featurization set.